ds3-autogen-go/src/main/kotlin/com/spectralogic/ds3autogen/go/generators/request/BaseRequestGenerator.kt

```diff
@@ -22,7 +22,10 @@ import com.spectralogic.ds3autogen.api.models.apispec.Ds3Param
 import com.spectralogic.ds3autogen.api.models.apispec.Ds3Request
 import com.spectralogic.ds3autogen.api.models.enums.Operation
 import com.spectralogic.ds3autogen.api.models.enums.Requirement
+import com.spectralogic.ds3autogen.go.generators.client.toRequestPath
```

Comment:
You added imports here, but I don't see these new imports being used.  Is this due to moving some code our of the package?

Reply:
Exactly. All new imports are stuff I moved from the request to the client. I just double checked and all imports are used.